src/Microsoft.HttpRepl.IntegrationTests/Commands/PrefCommandTests.cs

```diff
@@ -0,0 +1,217 @@
+using System;
```

Comment:
nit: add copyright comments to the top of any new files (or files which didn't have it already)

Reply:
Saw your comments on Bhavya's PR so the commit just above this comment (79435d7) added them to the new files in this PR.